app/views/state_file/questions/az_incarcerated/edit.html.erb

```diff
@@ -1,17 +1,15 @@
-<% title = (current_intake.filing_status_mfj? || current_intake.filing_status_mfs?) ?
-             t(".title_spouse", tax_year: MultiTenantService.statefile.current_tax_year) :
-             t(".title_single", tax_year: MultiTenantService.statefile.current_tax_year) %>
+<% title = t(".title", tax_year: MultiTenantService.statefile.current_tax_year, count: current_intake.filer_count) %>
 
 <% content_for :page_title, title %>
 <% content_for :card do %>
-  <h1 class="h2" id="main-question"><%=title %></h1>
+  <h1 class="h2"><%= title %></h1>
 
   <%= form_with model: @form, url: { action: :update }, local: true, method: "put", builder: VitaMinFormBuilder do |f| %>
     <div class="blue-group">
-      <%= f.cfa_radio_set(:was_incarcerated, collection: [
+      <%= f.cfa_radio_set(:was_incarcerated, label_text: title, collection: [
         { value: "yes", label: t("general.affirmative") },
         { value: "no", label: t("general.negative") },
-      ]) %>
+      ], legend_class: "sr-only") %>
```

Comment:
Is this some general consistency clean-up?

Reply:
@mpidcock good question! yeah this was something I noticed on some pages included `id="main-question"` in the main header. I wasn't sure what this was referencing so I called Ben about it. But basically we concluded that these `main-question` ids were not referencing anything without an `aria-describedby` being used anywhere. Since there wasn't a param field where we could feed an `aria-describedby` into the `cfa_radio_set` item we decided to add the label text to the radio set with "screen reader only" class that would allow the screen reader to associate it to the radio options. Ideally we would do an accessibility check and check the screenreader could read the whole app in the intended way eventually but not sure if this will happen in the near future so just opting to change them as I come across the pages for now.